README.md

```diff
@@ -82,7 +82,7 @@ Below are the environment needed specifically for CCD domain.
 ### 3.7) Run BEFTA Framework With Gradle
 1. Install Gradle 4.1 or higher. You can simply copy a gradle wrapper from `https://github.com/hmcts/befta-fw`.
 2. Add the following dependency to your build.gradle file:  
-   `testCompile group: 'uk.gov.hmcts', name: 'befta-fw', version: '1.2.1'`
+   `testCompile group: 'com.github.hmcts', name: 'befta-fw', version: '6.13.4'`
```

Comment:
Have to change the group?

Reply:
Yep, the HMCTS organisation has been set up to use the group `com.github.hmcts` on Jitpack.